core/indigo-core/molecule/src/molfile_saver.cpp

```diff
@@ -149,6 +149,7 @@ void MolfileSaver::_saveMolecule(BaseMolecule& mol, bool query)
 
     MoleculeCIPCalculator mcc;
     mcc.updateCIPStereoDescriptors(mol, add_stereo_desc);
+    mcc.addCIPSgroups(mol);
```

Comment:
s-group should be added only if option is on

Reply:
updateCIPStereoDescriptors adds corresponding properties to bonds and atoms if add_stereo_desc = true.
addCIPSgroups has not effect if no properties has been added.